### ISSUE TITLE
6.0 snapshots build fail

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#19183]
### Describe the bug

```
1) PhpMyAdmin\Tests\HeaderTest::testEnable
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
     'allow_third_party_framing' => false,
     'base_dir' => '',
     'theme_path' => '',
-    'version' => 'v=6.0.0-dev+20240528.1f6755b0d2',
+    'version' => 'v=6.0.0-dev%2B20240528.1f6755b0d2',
     'text_dir' => 'ltr',
     'server' => 0,
     'title' => 'phpMyAdmin',

/home/runner/work/phpmyadmin/phpmyadmin/release/phpMyAdmin-6.0+snapshot/tests/unit/HeaderTest.php:113
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/phpunit/phpunit/src/TextUI/Application.php:199
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/phpunit/phpunit/phpunit:104
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/bin/phpunit:122

2) PhpMyAdmin\Tests\ResponseRendererTest::testHtmlResponse
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
   <link rel="icon" href="favicon.ico" type="image/x-icon">
   <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
   <link rel="stylesheet" type="text/css" href="/jquery/jquery-ui.css">
-  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/lib/codemirror.css?v=6.0.0-dev+20240528.1f6755b0d2">
-  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/addon/hint/show-hint.css?v=6.0.0-dev+20240528.1f6755b0d2">
-  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/addon/lint/lint.css?v=6.0.0-dev+20240528.1f6755b0d2">
-  <link rel="stylesheet" type="text/css" href="/css/theme.css?v=6.0.0-dev+20240528.1f6755b0d2">
+  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/lib/codemirror.css?v=6.0.0-dev%2B20240528.1f6755b0d2">
+  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/addon/hint/show-hint.css?v=6.0.0-dev%2B20240528.1f6755b0d2">
+  <link rel="stylesheet" type="text/css" href="js/vendor/codemirror/addon/lint/lint.css?v=6.0.0-dev%2B20240528.1f6755b0d2">
+  <link rel="stylesheet" type="text/css" href="/css/theme.css?v=6.0.0-dev%2B20240528.1f6755b0d2">
   <title>phpMyAdmin</title>
     <script data-cfasync="false" src="js/dist/runtime.js?v=6.0.0-dev%2B20240528.1f6755b0d2"></script>
   <script data-cfasync="false" src="js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240528.1f6755b0d2"></script>

/home/runner/work/phpmyadmin/phpmyadmin/release/phpMyAdmin-6.0+snapshot/tests/unit/ResponseRendererTest.php:181
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/phpunit/phpunit/src/TextUI/Application.php:199
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/phpunit/phpunit/phpunit:104
/tmp/phpMyAdmin-phpunit.zbqq4Skmk/vendor/bin/phpunit:122
```

### To Reproduce

1. Go to https://github.com/phpmyadmin/phpmyadmin/actions/runs/9261475004/job/25476897187
2. See error

### Expected behavior

Nightly builds should work.

**Comment by liviuconcioiu** [comment]
Also:

` resources/po/ko.po:2486: 'msgstr[0]' is not a valid PHP format string, unlike 'msgid_plural'. Reason: In the directive number 2, the character '<' is not a valid conversion specifier.`

https://github.com/phpmyadmin/phpmyadmin/actions/runs/9310796920/job/25628870869

**Comment by MauricioFauth** [comment]
Fixed by #19227
